README.rst

```diff
@@ -273,3 +368,10 @@ recommend the following workflow to pull in future changes:
 .. _MikroE Weather Click: https://www.mikroe.com/weather-click
 .. _MikroE HVAC Click: https://www.mikroe.com/hvac-click
 .. _Reference Design Template: https://github.com/golioth/reference-design-template
+.. _Air Quality Monitor Project Page: https://projects.golioth.io/reference-designs/air-quality-monitor
+.. _nRF9160 DK Follow-Along Guide: https://example.com
```

Comment:
@cdwilson update this with the final link after https://github.com/golioth/projects/pull/40 is merged